proj/donut-accel/src/donut.c

```diff
@@ -0,0 +1,87 @@
+// The donut code with fixed-point arithmetic; no sines, cosines, square roots, or anything.
+// a1k0n 2020
+// Code from: https://gist.github.com/a1k0n/80f48aa8911fffd805316b8ba8f48e83
+// For more info:
+// - https://www.a1k0n.net/2011/07/20/donut-math.html
+// - https://www.youtube.com/watch?v=DEqXNfs_HhY
+
```

Comment:
Since this is the accelerated version maybe it would be nice if we like make the accelerated instructions standout more with caps (or a comment or something)? Just thinking at like HPCA if we walk thru this code (or if anyone is walking thru this as a tutorial example) it would be nice for people to easily see what instructions are the accelerated ones by glance.

Reply:
I've capitalized the custom macros.